temporal_capi/src/duration.rs

```diff
@@ -56,42 +52,6 @@ pub mod ffi {
         }
     }
 
-    impl TimeDuration {
```

Comment:
note: TimeDuration is not used in V8 right now, removing this API will not need complex migration